api.swagger.yaml

```diff
@@ -1332,6 +1332,49 @@ paths:
             schema:
               $ref: '#/components/schemas/post-schema-create'
         required: true
+  /posts/list:
+    get:
+      summary: List of posts.
+      tags:
+        - posts
+      parameters:
+        - description: Field to sort by
+          in: query
+          name: sort
```

Comment:
No default and not required - so what do we sort by if it is omitted?

Reply:
update